draft-barnes-sframe-mls.md

```diff
@@ -102,11 +140,22 @@ SFrame uses signatures, these are the keys used to generate SFrame signatures.
 
 # Security Considerations
 
-TODO
+The security properties provided by MLS are discussed in detail in
+[!I-D.ietf-mls-arch] and [!I-D.ietf-mls-protocol].  This document extends those
+guarantees to SFrame.
+
+It should be noted that the per-sender keys derived here do not provide
+per-sender authentication, since any member of the group could derive the same
+keys (as indeed they must in order to decrypt the protected payload).
+Per-sender keys are derived only to avoid nonce collision among multiple
+unsynchronized senders.  So the authentication limitations of SFrame remain:
+There is per-sender authentication only when signatures are used.  Otherwise,
+SFrame only authenticates membership in the group, and members are free to
+impersonate each other.
 
 # IANA Considerations
 
-TODO
+This document makes no request of IANA.
```

Comment:
Where do we define ciphersuites? Is that part of SFrame?

Reply:
Yeah, the ciphersuites are defined in the base SFrame document.